client/styles/sc-text-styles.html

```diff
@@ -347,21 +347,19 @@
         color: var(--sc-paper-tooltip-text-color);
       }
 
-      .varnote td,
-      .varnote li,
-      .varnote ul {
+      .varnote td {
         padding: 0;
         vertical-align: middle;
         border: none;
       }
 
       .varnote td:first-child:not(:last-child) {
         font-weight: bold;
+        padding-right: 4px;
```

Comment:
please use `var(--sc-size-xs)` instead of `4px`. These constants are defined in `sc-utility-styles.html` to keep using consistent pixel values throughout the code.